oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst

```diff
@@ -24,6 +24,8 @@ OEP-52: Event Bus Architecture
      - 2021-08-16
    * - Review Period
      - 2022-06-09 - 2022-06-30
+   * - References
+     - Follow up work: `event bus project roadmap`_
```

Comment:
If we point here, we should have the roadmap point to the issue to determine what next steps are specifically required for getting to an accepted state.

Reply:
I'm worried about going around in circles with the issue and the OEP pointing to each other. My inclination is to leave this PR unmerged and the arch-bom issue in blocked until we have a clearer path laid out in that issue so we know exactly what to put here instead of having the follow up work be to determine the follow up work.

Reply:
Updated idea:
1. Don't update the roadmap, and let the new issue be about how to get this to accepted be unrelated to this change.
2. Merge this PR and close your current ticket.
Thoughts?

Reply:
fine by me!